### DISCUSSION TITLE
Custom SSL

### DISCUSSION THREAD

**Discussion by imangm** [vercel/community#711] [Help] [answered]
Hi 

Does anyone know how I can assign an EV SSL for my website on Vercel? By default, Vercel generates a let's encrypt SSL but I need to use a custom SSL. I didn't find any doc that explains it. Is it even possible?

Thanks.
Iman

**Comment by amyegan** [comment, marked as answer]
Hi @imangm. Thanks for pointing out that this is missing from our documentation. We'll get that updated. Uploading custom SSL certificates is only available on our Enterprise plans for now.
